### PR TITLE
#684- Conversion from Trust/Observation to EigenTrust's Opinions/Experiences

### PR DESCRIPTION
**Pull request by kpudlik**

### CODE REVIEW

File: src/main/scala/org/constellation/trust/EigenTrust.scala [hunk 1]
```diff
@@ -23,14 +26,14 @@ object EigenTrust {
   nodesWithEdges.foreach { node: TrustNode =>
     node.edges.foreach { edge =>
       val trust = edge.trust / 2 + 0.5 // Revert from -1 to 1 => 0 to 1
-//      println(trust)
+      //      println(trust)
```
Comment (mchrapek) [inline src/main/scala/org/constellation/trust/EigenTrust.scala]: probably should be removed

File: src/main/scala/org/constellation/trust/EigenTrust.scala [hunk 2]
```diff
@@ -41,6 +44,47 @@ object EigenTrust {
 
   val trustMap: Map[Integer, lang.Double] = eigenTrust.getTrust(0).asScala.toMap
 
-//  trustMap.toSeq.sortBy(_._1).foreach { println }
+  //  trustMap.toSeq.sortBy(_._1).foreach { println }
```
Comment (mchrapek) [inline src/main/scala/org/constellation/trust/EigenTrust.scala]: probably should be removed before merge

File: src/main/scala/org/constellation/trust/EigenTrust.scala (hunk 1 above)
Comment (kpudlik) [reply src/main/scala/org/constellation/trust/EigenTrust.scala]: Most of the code inside EigenTrust will be probably removed but for now let's leave it. We will clean that object anyway after switching the logic

File: src/main/scala/org/constellation/trust/EigenTrust.scala (hunk 2 above)
Comment (kpudlik) [reply src/main/scala/org/constellation/trust/EigenTrust.scala]: As above

**Pull request merged**